ConsentViewController/Classes/SPError.swift

```diff
@@ -256,6 +256,14 @@ import Foundation
     public override var campaignType: SPCampaignType { get { .gdpr } set {} }
 }
 
+@objcMembers public class InvalidMetaDataQueryParamsError: SPError {
+    override public var spCode: String { "sp_metric_invalid_meta_data_query_params" }
+}
+
+@objcMembers public class InvalidMetaDataQueryResponseError: SPError {
```

Comment:
I believe the correct name should be `InvalidMetaDataResponseError` (there's no _query_ in the response).